spec/environment/pi.rb

```diff
@@ -0,0 +1,22 @@
+# encoding: UTF-8
+#
+# Copyright (c) 2010-2017 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+module GoodData
+  module Environment
+    module ConnectionHelper
+      set_const :DEFAULT_DOMAIN, 'default'
+      set_const :DEFAULT_SERVER, 'https://js-devel.na.intgdc.com'
```

Comment:
We should parametrise the hostname and everything env-specific so that we can run this with any PI and plug into CI easily.